packages/big-design/src/components/StatelessPagination/StatelessPagination.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  ArrowDropDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@bigcommerce/big-design-icons';
+import React, { memo } from 'react';
+
+import { MarginProps } from '../../helpers';
+import { Dropdown, DropdownItem } from '../Dropdown';
+import { Flex, FlexItem } from '../Flex';
+
+import { StyledButton } from './styled';
+
+export interface StatelessPaginationLocalization {
+  previousPage: string;
+  nextPage: string;
+}
+
+const defaultLocalization: StatelessPaginationLocalization = {
+  previousPage: 'Previous page',
+  nextPage: 'Next page',
+};
+
+export interface StatelessPaginationProps extends MarginProps {
+  itemsPerPage: number;
+  itemsPerPageOptions: number[];
+  onItemsPerPageChange(range: number): void;
```

Comment:
🤔 is this a `range?` or more like a `count`?

Reply:
I guess not really, I'd think of a range as a `1 -10`, `50-100`, two value concept

to me, this'd be `newItemsPerPage` or `chosenItemsPerPage` (given it has been picked by the user, but not yet added to state)

but they're long and we already have precedent of `range` on the docs for regular pagination: https://bigcommerce.github.io/big-design/pagination#props